YandexCheckoutPaymentsApi/Source/Models/ConfirmationType.swift

```diff
@@ -27,68 +27,12 @@ import Foundation
 /// Type of custom payment confirmation process.
 /// Read more about the scenarios of [confirmation of payment](https://kassa.yandex.ru/docs/guides/#confirmation)
 /// by the buyer.
-public struct ConfirmationType: OptionSet, Codable {
-
-    public private(set) var rawValue: Set<String>
+public enum ConfirmationType: String, Codable {
```

Comment:
Please, tell about these changes.
Using a set of options in this case is preferable.

Reply:
We need to use this model in single value as parameter
fxd in https://github.com/yandex-money/yandex-checkout-payments-api-swift/pull/7